app/controllers/declaration/DepartureTransportController.scala

```diff
@@ -71,23 +71,26 @@ class DepartureTransportController @Inject() (
   }
 
   private def submit(fun: () => Future[Result])(implicit request: JourneyRequest[AnyContent]): Future[Result] = {
-    val declaration = request.cacheModel
-    val isPostalOrFTI =
-      isPostalOrFTIModeOfTransport(declaration.transportLeavingBorderCode) ||
-        declaration.`type` != CLEARANCE && isPostalOrFTIModeOfTransport(declaration.inlandModeOfTransportCode)
-    val resetValueAndGotoNextPage = isPostalOrFTI || isGuernseyOrJerseyDestination(declaration)
-    if (resetValueAndGotoNextPage) updateCache(DepartureTransport(None, None)).map(_ => nextPage) else fun()
+    val resetValueAndGotoNextPage = isPostalOrFTI || isGuernseyOrJerseyDestination(request.cacheModel)
+    if (resetValueAndGotoNextPage || isSimplifiedOrOccasional) updateCache(DepartureTransport(None, None)).map(_ => nextPage)
+    else fun()
   }
 
-  private def nextPage(implicit request: JourneyRequest[AnyContent]): Result = {
-    val call =
+  private def nextPage(implicit request: JourneyRequest[AnyContent]): Result =
+    navigator.continueTo {
       if (request.declarationType == CLEARANCE) ExpressConsignmentController.displayPage
-      else if (request.cacheModel.isInlandOrBorder(Border)) TransportCountryController.displayPage
+      else if (request.cacheModel.isInlandOrBorder(Border) || (isSimplifiedOrOccasional && !isPostalOrFTI)) TransportCountryController.displayPage
```

Comment:
I think you do not need to change this. It should just be as it was before...
```
else if (request.cacheModel.isInlandOrBorder(Border)) TransportCountryController.displayPage
```

Reply:
Do you mean to remove the `(isSimplifiedOrOccasional && !isPostalOrFTI)` condition here?